File: ucare_frontend/src/service/diagnosisService.js
import axios from 'axios';
import { DIAGNOSIS_API_BASE_URL } from './urlConfig'

class diagnosisService {
  retrieveByPatientNo(patientNo){
    return axios.get(DIAGNOSIS_API_BASE_URL + '/retrieveByPatientNo/' + patientNo);
  }
}

export default new diagnosisService();

File: ucare_backend/src/main/java/com/douzone/ucare/service/DiagnosisService.java
package com.douzone.ucare.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.douzone.ucare.repository.DiagnosisRepository;
import com.douzone.ucare.vo.DiagnosisVo;

@Service
public class DiagnosisService {
	
	@Autowired
	private DiagnosisRepository DiagnosisRepository;
	
	public List<DiagnosisVo> retrieveByPatientNo(Long patientNo) {
		return DiagnosisRepository.retrieveByPatientNo(patientNo);
	}
}
